admin-site/src/Button.svelte

```diff
@@ -0,0 +1,52 @@
+<script>
+export let buttonText = "";
+export let buttonType = 'btn-outline-primary btn-block';
+export let dataTarget = undefined;
+export let value = "";
+export let disabled = undefined;
+export let bsSizePosition = "col-md-12";
```

Comment:
Unclear prop name...

Reply:
the bsSizePosition?....yeah.. bootstrap Size and Position, not great. grid? gridLayout?

Reply:
bootstrapGridStyle?